Makefile

```diff
@@ -300,24 +308,24 @@ test-benchmark: ## Run the benchmark tests -- Note that this can only be ran for
 e2e: namespace tear-down operator-sdk image-to-cluster openshift-user deploy-crds ## Run the end-to-end tests
 	@echo "WARNING: This will temporarily modify deploy/operator.yaml"
 	@echo "Replacing workload references in deploy/operator.yaml"
-	@$(SED) 's%$(IMAGE_REPO)/$(RELATED_IMAGE_OPENSCAP_NAME):$(RELATED_IMAGE_OPENSCAP_TAG)%$(RELATED_IMAGE_OPENSCAP_PATH)%' deploy/operator.yaml
-	@$(SED) 's%$(IMAGE_REPO)/$(APP_NAME):latest%$(RELATED_IMAGE_OPERATOR_PATH)%' deploy/operator.yaml
+	@$(SED) 's%$(DEFAULT_IMAGE_OPENSCAP_PATH)%$(RELATED_IMAGE_OPENSCAP_PATH):$(RELATED_IMAGE_OPENSCAP_TAG)%' deploy/operator.yaml
```

Comment:
Hmm, I wonder if this is the reason for failure I'm seeing..while running the e2e tests locally I see the tag twice in the scanner pods:
```
  Warning  InspectFailed   5s (x4 over 22s)  kubelet            Failed to apply default image tag "image-registry.openshift-image-registry.svc:5000/openshift/openscap-ocp:1.3.5:1.3.5": couldn't parse image reference "image-registry.openshift-image-registry.svc:5000/openshift/openscap-ocp:1.3.5:1.3.5": invalid reference format
```
I guess this might also be the reason for the CI failing. I'm sorry I didn't have enough time to dive deeper into why this is failing..